scripts/release-update.sh

```diff
@@ -12,29 +12,29 @@ set -eu
 CRYSTAL_VERSION=$1
 
 # Edit PREVIOUS_CRYSTAL_BASE_URL in .circleci/config.yml
-sed -i -E "s|[0-9.]+/crystal-[0-9.]+-[0-9]|$CRYSTAL_VERSION/crystal-$CRYSTAL_VERSION-1|g" .circleci/config.yml
+sed -i '' -E "s|[0-9.]+/crystal-[0-9.]+-[0-9]|$CRYSTAL_VERSION/crystal-$CRYSTAL_VERSION-1|g" .circleci/config.yml
```

Comment:
Is this script only run on MacOS?  `-i ''` doesn't work on Linux

Reply:
FWIW, this particular thing is the bane of my existence in my other projects.  In those other projects I do `brew install coreutils` to get the gnu versions of common tools, then temporarily make `sed` point to `gsed` before running any scripts.  Trying to make them work on both OSes can be a nightmare

Reply:
yikes, thanks @Fryguy

Reply:
If it helps, you can determine if MacOS using the following test in bash: `if [[ "$OSTYPE" == "darwin"* ]]; then ...`

Reply:
I thought about it, but I decided to test `sed` directly in case it's locally linked to gnu's